src/main/scala/ru/otus/sc/game/model/Action.scala

```diff
@@ -0,0 +1,73 @@
+package ru.otus.sc.game.model
+
+/**
+  * Игровое дейтсвие
+  */
+trait Action {
+
+  /**
+    * Название
+    *
+    * @return название
+    */
+  def name(): String
```

Comment:
```suggestion
  def name: String
```

И во многих последующих методах.